reliability-engineering/terraform/modules/concourse-monitoring/lb.tf

```
@@ -0,0 +1,103 @@
+resource "aws_lb" "concourse_monitoring" {
+  name               = "${var.deployment}-concourse-monitoring"
+  internal           = false
+  load_balancer_type = "application"
+  security_groups    = ["${aws_security_group.concourse_monitoring_lb.id}"]
+  subnets            = ["${var.public_subnet_ids}"]
+
+  tags = {
+    Name       = "${var.deployment}-concourse-monitoring"
+    Deployment = "${var.deployment}"
+  }
+}
+
+resource "aws_lb_listener" "concourse_monitoring_https" {
+  load_balancer_arn = "${aws_lb.concourse_monitoring.arn}"
+  port              = "443"
+  protocol          = "HTTPS"
+  ssl_policy        = "ELBSecurityPolicy-2015-05"
+  certificate_arn   = "${aws_acm_certificate.concourse_monitoring.arn}"
+
+  default_action {
+    type = "fixed-response"
+
+    fixed_response {
+      content_type = "text/plain"
```

```suggestion
      content_type = "text/plain; charset=utf-8"
```
don't want mojibake to ruin the emoji 😁